test_compute_cdftt2000 failure

The following hypothesis example is failing:

```python
=================================== FAILURES ===================================

____________________________ test_compute_cdftt2000 ____________________________

>   ???

tests/test_epochs.py:166: 

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

dtime = datetime.datetime(2292, 5, 1, 0, 0)

    @given(random_dtime)

    @settings(max_examples=100)

    @example(datetime(1972, 1, 1, 0, 0))

    def test_compute_cdftt2000(dtime):

        random_time = [dtime.year, dtime.month, dtime.day,

                       dtime.hour, dtime.minute, dtime.second,

                       dtime.microsecond // 1000,  # Millisecond

                       randint(0, 999),     # Microsecond

                       randint(0, 999),     # Nanosecond

                       ]

        x = cdfepoch.breakdown(cdfepoch.compute(random_time))

        for i, t in enumerate(x):

>           assert t == random_time[i], f'Time {random_time} was not equal to {x}'

E           AssertionError: Time [2292, 5, 1, 0, 0, 0, 0, 864, 394] was not equal to [1707, 10, 12, 0, 26, 3, 291, 312, 778]

E           assert 1707 == 2292

E             +1707

E             -2292

tests/test_epochs.py:177: AssertionError
```

--- a/tests/test_epochs.py
+++ b/tests/test_epochs.py
@@ -31,6 +31,13 @@ Each of these results were hand checked using either IDL or
 other online resources.
 '''
 
+# These are the supported years for TT2000 dates; see
+# https://cdf.gsfc.nasa.gov/html/leapseconds_requirements.html
+# section 2.1
+random_tt2000_dtime = strategies.datetimes(
+    min_value=datetime(1707, 9, 22, 12, 13, 16),
+    max_value=(datetime(2292, 4, 11, 11, 46, 8)))
+
 # These are the supported years for CDF files; see
 # https://spdf.gsfc.nasa.gov/pub/software/cdf/doc/cdf371/cdf371ug.pdf
 # page 55
@@ -162,7 +169,7 @@ def test_compute_cdfepoch16(dtime):
         i += 1
 
 
-@given(random_dtime)
+@given(random_tt2000_dtime)
 @settings(max_examples=100)
 @example(datetime(1972, 1, 1, 0, 0))
 def test_compute_cdftt2000(dtime):

